File: source/includes/code/node/crud-insert-final-query.js
const { MongoClient } = require("mongodb");
// Replace the uri string with your MongoDB deployment's connection string.
const uri =
"mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&writeConcern=majority";

const client = new MongoClient(uri);

async function run() {
  try {
    await client.connect();
    // database and collection code goes here
    const db = client.db("sample_guides");
    const coll = db.collection("comets");

    // insert code goes here
    const docs = [
      {Name: "Halley's Comet", OfficialName: "1P/Halley", OrbitalPeriod: 75, Radius: 3.4175, Mass: 2.2e14},
      {Name: "Wild2", OfficialName: "81P/Wild", OrbitalPeriod: 6.41, Radius: 1.5534, Mass: 2.3e13},
      {Name: "Comet Hyakutake", OfficialName: "C/1996 B2", OrbitalPeriod: 17000, Radius: 0.77671, Mass: 8.8e12}
    ];

    const result = await coll.insertMany(docs);

    // display insert ids code goes here
    console.log(result.insertedIds);

  } finally {
    // Ensures that the client will close when you finish/error
    await client.close();
  }
}
run().catch(console.dir);

File: source/includes/code/python/crud_insert_final_query.py
from pymongo import MongoClient

# Replace the uri string with your MongoDB deployment's connection string.
uri = "mongodb+srv://<user>:<password>@<cluster-url>?retryWrites=true&writeConcern=majority"

client = MongoClient(uri)

# database and collection code goes here
db = client.sample_guides
coll = db.comets

coll.drop()

# insert code goes here
docs = [
    {"name": "Halley's Comet", "officialName": "1P/Halley", "orbitalPeriod": 75, "radius": 3.4175, "mass": 2.2e14},
	{"name": "Wild2", "officialName": "81P/Wild", "orbitalPeriod": 6.41, "radius": 1.5534, "mass": 2.3e13},
	{"name": "Comet Hyakutake", "officialName": "C/1996 B2", "orbitalPeriod": 17000, "radius": 0.77671, "mass": 8.8e12},
    ]

result = coll.insert_many(docs)

# display insert ids code goes here
print(result.inserted_ids)

# Close the connection to MongoDB when you're done.
client.close()
